sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -1,20 +1,19 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 12.0.0-beta.5 (2020-11-13)
 
 ### Breaking Changes
```

Comment:
No new features or bug fixes this release?

Reply:
True, only test changes and breaking chages.